src/test/compile-fail/attr-usage-repr.rs

```diff
@@ -25,19 +25,19 @@ struct SPacked(f64, f64);
 struct SSimd(f64, f64);
 
 #[repr(i8)] //~ ERROR: attribute should be applied to enum
-struct SInt(f64, f64);
+struct SInt(f64, f64); //~ not an enum
 
 #[repr(C)]
 enum EExtern { A, B }
 
 #[repr(align(8))] //~ ERROR: attribute should be applied to struct
-enum EAlign { A, B }
+enum EAlign { A, B } // not a struct
 
```

Comment:
Oops, just noticed these are missing tildes (but the test still passes?)

Reply:
surveying nearby examples it seems I simply shouldn't include the "note" messages in this test anyways? (we just want to know that the error is triggered here?)

Reply:
@ExpHP if you run `build/<ENV>/stage1/bin/rustc src/test/compile-fail/attr-usage-repr.rs` you will see the actual output. The test as it was previously was incorrect, as it wasn't asserting the label in that line. It should have been a `//~ NOTE: not a struct` there. Please change it here and elsewhere to avoid regressions and actually test this change.